storage/src/mmr/mod.rs

```diff
@@ -41,6 +43,8 @@
 //!      1     2     5      9     12     17
 //!           / \   / \    / \   /  \   /  \
 //!      0   0   1 3   4  7   8 10  11 15  16 18
+//!
+//! number   0   1 2   3  4   5  6   7  8   9 10
```

Comment:
nit: to keep consistent style recommend `Number`

Reply:
done